packages/vite/src/fully-react/find-styles.ts

```diff
@@ -3,15 +3,15 @@ import path from 'node:path'
 import type { ModuleNode, ViteDevServer } from 'vite'
 
 async function find_deps(
-  vite: import('vite').ViteDevServer,
-  node: import('vite').ModuleNode,
-  deps: Set<import('vite').ModuleNode>
+  vite: ViteDevServer,
+  node: ModuleNode,
+  deps: Set<ModuleNode>
```

Comment:
@Tobbe is there a reason to use `import('vite').` here in this file?

Reply:
_gentle bump to @Tobbe - this is a fairly broad PR so the longer it stays open, the more painful the merge conflicts can become._

Reply:
This is code I mostly copy/pasted from another project. Looks like `import type { ModuleNode } from 'vite'` should take care of the type here. Your change looks good

Reply:
thanks for double checking!